pkgs/tools/security/signify/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, libbsd, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "signify-${version}";
+  version = "v24";
```

Comment:
Note that the `v24` will be interpreted as part of the package name and the version will in fact be empty.  See `builtins.parseDrvName` for how names are parsed.

Reply:
I'm not sure I follow. From the docs:
```
*builtins.parseDrvName s*

    Split the string s into a package name and version. The package name is everything up to but not including the first dash followed by a digit, and the version is everything following that dash. The result is returned in a set { name, version }. Thus, builtins.parseDrvName "nix-0.12pre12876" returns { name = "nix"; version = "0.12pre12876"; }.
```

Reply:
Ah yes, the v...

Reply:
Fixed.